Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
Hmm, that's strange, it should be a dependency, but it shouldn't have to reinstall every time, it shouldn't even have to do anything if the venv is updated.